sre-recipes/sandboxctl

```diff
@@ -200,7 +200,7 @@ def destroy():
     """Delete an existing sandbox"""
     os.chdir(os.path.abspath(sys.path[0]))
     destroy_command = "../terraform/destroy.sh"
-    Recipe._run_command(destroy_command)
+    Recipe._run_command_interactive(destroy_command)
```

Comment:
Could you please open a new issue for refactoring sandboxctl (unrelated to your changes).
It looks weird now that sandboxctl is hidden in sre-recipes folder. We originally meant that it will only be used for recipes but since then we extended it to support automated cluster creation, destroy and other things.

We need to move sandboxctl to its own folder under root of the repo. Maybe it would make sense to make it modular so that sre recipe commands will stay in sre-recipes folder while sandbox-wide commands will be under [root]/sandboxctl folder.

Reply:
Opened [issue/748](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/issues/748)